src/components/templates/SeriesListPage.tsx

```diff
@@ -1,18 +1,22 @@
-import { allSeriesInfo } from '../../assets/pages/series/lowPheLife'
+import { useTheme } from '@mui/material'
+import allSeriesInfo from '../../assets/pages/series'
 import { PageTitle } from '../atoms/pageTitle/PageTitle'
 import { SeriesList } from '../organisms/seriesList/SeriesList'
 
 export function SeriesListPage() {
+  const theme = useTheme()
+  const barColor = theme.palette.series.main
   const seriesListSubheaderText =
     'Series List - Risus velit lectus ut vitae sit nunc mauris mattis erat. Posuere adipiscing nisi lacinia lacinia facilisi venenatis aliquam.'
 
   return (
     <>
       <PageTitle
         title="Series"
+        titleUnderlineColor={barColor}
```

Comment:
Why not just use `theme.pallete.series.main` here? Why set the variable?

Reply:
Dunno. Could have. Might have been thinking of doing something more.